packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -41,16 +40,8 @@ const selectors = {
 	// Settings panel.
 	settingsPanel: '.interface-complementary-area',
 
-	// Publish panel (including post-publish)
-	publishPanel: '.editor-post-publish-panel',
-	// With the selector below, we're targeting both "View Post" buttons: the one
-	// in the post-publish pane, and the one that pops up in the bottom-left
-	// corner. This addresses the bug where the post-publish panel is immediately
-	// closed when publishing with certain blocks on the editor canvas.
-	// See https://github.com/Automattic/wp-calypso/issues/54421.
-	viewButton: 'a:text-matches("View (Post|Page)", "i")',
-	addNewButton: '.editor-post-publish-panel a:text-matches("Add a New P(ost|age)")',
-	closePublishPanel: 'button[aria-label="Close panel"]',
+	// Toast
+	toastViewPostLink: '.components-snackbar__content a:text-matches("View (Post|Page)", "i")',
```

Comment:
Now that the `EditorPublishPanelComponent` has taken over interactions with the `View Post/Page` button that appears after the article is published, the main `GutenbergEditorPage` should only handle the toast, hence the rename and narrowing of the selector.